data/json/items/containers.json

```diff
@@ -203,10 +222,13 @@
     "pocket_data": [
       {
         "pocket_type": "CONTAINER",
+        "min_item_volume": "2 ml",
```

Comment:
not sure what this is for; watertight actually implies the item has no minimum item volume.